diff --git a/vold.fstab b/vold.fstab
index d904599..5ff4378 100644
--- a/vold.fstab
+++ b/vold.fstab
@@ -25,4 +25,6 @@
 # OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN
 # IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
 
-dev_mount sdcard /mnt/sdcard auto /devices/platform/msm_sdcc.1/mmc_host /devices/platform/msm_sdcc.3/mmc_host
+#dev_mount sdcard /mnt/sdcard auto /devices/platform/msm_sdcc.1/mmc_host
+dev_mount sdcard /mnt/sdcard auto /devices/platform/msm_sdcc.3/mmc_host
+dev_mount extsdcard /mnt/extsdcard auto /devices/platform/msm_sdcc.1/mmc_host
